PKGBUILD

```diff
@@ -6,15 +6,17 @@
 # Contributor: Victor Hugo Souza <vhbsouza@gmail.com>
 
 pkgname=gitkraken
-pkgrel=1
+pkgrel=2
 pkgver=2.5.0
+nodegitver=0.18.3
+electronver=1.2.8
 pkgdesc="The intuitive, fast, and beautiful cross-platform Git client."
-url="http://www.gitkraken.com/"
+url="https://www.gitkraken.com/"
```

Comment:
`https` is better
![image](https://cloud.githubusercontent.com/assets/18448635/25882164/eccd8cf4-355a-11e7-9068-2393ba95b469.png)